res/push-to-core/configuration.toml

```diff
@@ -35,3 +35,21 @@ AccessTokenFile = '/tmp/edgex/secrets/AppService-push-to-core/consul-token' # ig
 
```

Comment:
need to remove `ConfigAccessTokenFile ` and `AccessTokenFile ` from the above config.

Reply:
Good catch. Thought I had got all those...